support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -442,125 +461,106 @@ function CheckoutComponent({
 	 * We might be able to defer this to the backend.
 	 */
 	let productFields: RegularPaymentRequest['product'];
-
-	if (productId === 'TierThree') {
```

Comment:
This was swapped out for a `switch` statement as [`switch` statements check for exhaustiveness, not `if` statements](https://github.com/microsoft/TypeScript/issues/38731)... 😿